app/models/reports/trainee_report.rb

```diff
@@ -365,6 +365,7 @@ def submitted_for_trn_at
     end
 
     def complete? = trainee.submission_ready?
+    alias_method :is_complete, :complete?
```

Comment:
```suggestion
    def is_complete = trainee.submission_ready?
```

Maybe... just being fussy...

Reply:
rubocop says no `Naming/PredicateName:`

Reply:
Could you just call it `completed`?

Reply:
yeah could do, just used what was asked in the ticket

Reply:
done